addOns/sequence/src/main/java/org/zaproxy/zap/extension/sequence/StdActiveScanRunner.java

```diff
@@ -156,12 +159,22 @@ public ZestResponse runStatement(
                     }
                 }
 
+                ExtensionAlert extAlert =
+                        Control.getSingleton()
+                                .getExtensionLoader()
+                                .getExtension(ExtensionAlert.class);
+                List<Alert> alerts =
+                        extAlert.getAllAlerts().stream()
```

Comment:
This should be done lazily, and it should not get all alerts, it should read only the ones that were raised.